code/modules/virus2/effect.dm

```diff
@@ -953,3 +953,41 @@ var/list/compatible_mobs = list(/mob/living/carbon/human)
 	stage = 1
 	activate(var/mob/living/carbon/mob,var/multiplier)
 		mob << "<span class = 'notice'> You feel optimistic!</span>"
+
+/datum/disease2/effect/anxiety
+	name = "Severe Anxiety"
+	stage = 1
```

Comment:
these probably shouldn't be stage 1---stage 1 is more "hey, you have something" (with few exceptions) rather than "oh gosh, bad things are happening already!"

Stage 2 would probably work though.